slam_toolbox_ws/README.md

```diff
@@ -0,0 +1,31 @@
+# ROS2 SLAM-TOOLBOX
+
+### Run with docker
+
+```bash
+git clone https://github.com/j3soon/ros2-agv-essentials.git
+```
+
+```bash
+cd ros2-agv-essentials/slam_toolbox_ws/docker
+docker-compose pull
+docker-compose up -d --build
+```
+
+### Simulation with gazebo
+
+- Attach to the container
+  ```sh
+  docker attach ros2-slam-toolbox-ws
+  ```
+
+- Run with the gazebo
+```sh
+ros2 launch slam_simulation slam_sim.launch.py
+```
+
+You can put something in the gazebo world, since currently is empty.
```

Comment:
Two suggestions here:
1. Use a default gazebo map such as [this](https://navigation.ros.org/tutorials/docs/navigation2_with_slam.html) or [this](https://roboticsbackend.com/ros2-nav2-generate-a-map-with-slam_toolbox/) for verifying that slam toolbox is indeed working as expected.
2. Add a short guide on using rqt to control the turtlebot for the ease of testing.